src/AAVE/Card/AssetsToBorrow.jsx

```diff
@@ -49,12 +73,31 @@ return (
         title: "Assets to borrow",
         body: (
           <>
+            {showAlert && (
+              <>
+                <Widget
+                  src={`${config.ownerId}/widget/AAVE.Card.Divider`}
+                  props={{ config }}
+                />
+                <AlertContainer>
+                  <img
+                    src={`${config.ipfsPrefix}/bafkreih3npgn6ydscd7mzjrxredamembxhlmdxnw5l4izpfru2rbucvdly`}
+                    width={16}
+                    height={16}
+                  />
+                  <div>
+                    To borrow you need to supply any asset to be used as
+                    collateral.
+                  </div>
+                </AlertContainer>
+              </>
+            )}
             {!debts || debts.length === 0 ? (
               <Widget
                 src={`${config.ownerId}/widget/AAVE.Card.CardEmpty`}
                 props={{
                   config,
-                  children: "Nothing borrowed yet",
+                  children: "Nothing borrows yet",
```

Comment:
borrowed or borrows? I think we just changed from `borrows` to `borrowed` in another PR: https://github.com/corndao/aave-v3-bos-app/pull/37/files